app/konnect/dev-portal/applications/application-overview.md

```diff
@@ -23,3 +23,49 @@ As an example, the Application can represent a mobile banking app and the Servic
 Generate Application credentials through the {{site.konnect_short_name}} Dev Portal in the Application detail page. The Application can have multiple credentials, or API keys. For more information about Application Credentials, refer to [Generate Credentials for an Application](/konnect/dev-portal/applications/dev-gen-creds/).
 
 In [konnect.konghq.com](https://konnect.konghq.com), admins can access a list of the installed authentication plugins via **Shared Config**. See [Enable Application Registration for a Service](/konnect/dev-portal/applications/enable-app-reg/) for more information about authentication flows.
+
+## Konnect-managed plugins
+
+When you enable application registration on a Service,
+{{site.konnect_saas}} enables two plugins automatically:
+[ACL](/hub/kong-inc/acl), and one of [Key Authentication](/hub/kong-inc/key-auth)
+or [OIDC](/hub/kong-inc/openid-connect). These plugins run in the background to
+support application registration for the Service and are managed by
+{{site.konnect_saas}}.
+
+Plugins enabled by app registration cannot be disabled or deleted using the toggle in
+the **Plugins** pane on the Services Version page. To disable or
+delete them, disable app registration.
```

Comment:
Might be clearer to lead with what to do, instead of what you can't do? IOW, something like "To disable or delete a plugin you enabled with app registration, you must disable app registration [itself?]. You cannot work with the toggle in the **Plugins** pane ..."